src/libopensc/pkcs15-prkey.c

```diff
@@ -582,6 +578,14 @@ sc_pkcs15_free_prkey(struct sc_pkcs15_prkey *key)
 	}
```

Comment:
please add `sc_mem_clear(key, sizeof *key);`. This may resolve a possible double free. (This is already present in `sc_pkcs15_erase_pubkey()`)